### PR TITLE
Add exponent settings to easing function page

### PR DESCRIPTION
**Pull request by HO-COOH**
## Description
Add exponent settings to easing function page

## Motivation and Context
Sometimes I just need to have a quick tryout to different values using this

## How Has This Been Tested?
Manual.

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUIGallery/ControlPages/EasingFunctionPage.xaml [hunk 1]
```diff
@@ -74,18 +74,25 @@
                     </Rectangle.RenderTransform>
                 </Rectangle>
             </Grid>
-
+            
+            <local:ControlExample.Options>
+                <NumberBox x:Name="AccelerateEasingExponent" Header="Exponent" AutomationProperties.Name="Accelerate easing exponent" Value="4.5"/>
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/EasingFunctionPage.xaml:79]: Recommend to include a min and max property

File: WinUIGallery/ControlPages/EasingFunctionPage.xaml [hunk 2]
```diff
@@ -74,18 +74,25 @@
...
+            </local:ControlExample.Options>
+            
             <local:ControlExample.Xaml>
                 <x:String xml:space="preserve">
 &lt;Storyboard x:Name="Storyboard2"&gt;
     &lt;DoubleAnimation Storyboard.TargetName="Translation" Storyboard.TargetProperty="X" From="0" To="200" &gt;
         &lt;DoubleAnimation.EasingFunction&gt;
-            &lt;ExponentialEase Exponent="4.5" EasingMode="EaseIn" /&gt;
+            &lt;ExponentialEase Exponent="$(AccelerateEasingExponent)" EasingMode="EaseIn" /&gt;
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/EasingFunctionPage.xaml:87]: This example is for xaml, but this looks like a cs binding?

File: WinUIGallery/ControlPages/EasingFunctionPage.xaml [hunk 3]
```diff
@@ -111,17 +118,25 @@
...
+                <NumberBox x:Name="DecelerateEasingExponent" Header="Exponent" AutomationProperties.Name="Decelerate easing exponent" Value="7"/>
+            </local:ControlExample.Options>
+            
             <local:ControlExample.Xaml>
                 <x:String xml:space="preserve">
 &lt;Storyboard x:Name="Storyboard3"&gt;
     &lt;DoubleAnimation Storyboard.TargetName="Translation" Storyboard.TargetProperty="X" From="0" To="200" &gt;
         &lt;DoubleAnimation.EasingFunction&gt;
-            &lt;ExponentialEase Exponent="7" EasingMode="EaseOut" /&gt;
+            &lt;ExponentialEase Exponent="$(DecelerateEasingExponent)" EasingMode="EaseOut" /&gt;
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/EasingFunctionPage.xaml:131]: Again: xaml or cs?

**Comment by Jay-o-Way** [comment]
Please show a screenshot with the code expanded

**Review by niels9001 — APPROVED** [review]
Tested, works well! Thanks!

**Comment by niels9001** [comment]
@ho-cooh I guess we need an empty commit here as well (as part of: https://github.com/microsoft/WinUI-Gallery/pull/1423#issuecomment-1889832276)

**Review by karkarl — APPROVED** [review]

**Comment by HO-COOH** [comment]
> @HO-COOH I guess we need an empty commit here as well (as part of: [#1423 (comment)](https://github.com/microsoft/WinUI-Gallery/pull/1423#issuecomment-1889832276))

Done

**Pull request merged**